fullCalendar/fullCalendar.d.ts

```diff
@@ -8,27 +8,6 @@
 
 declare namespace FullCalendar {
     export interface Calendar {
-
-        /**
-         * Formats a Date object into a string.
-         */
-        formatDate(date: Date, format: string, options?: Options): string;
-
-        /**
-         * Formats a date range (two Date objects) into a string.
-         */
-        formatDates(date1: Date, date2: Date, format: string, options?: Options): string;
-
-        /**
-         * Parses a string into a Date object.
-         */
-        parseDate(dateString: string, ignoreTimezone?: boolean): Date;
-
-        /**
-         * Parses an ISO8601 string into a Date object.
-         */
-        parseISO8601(dateString: string, ignoreTimezone?: boolean): Date;
```

Comment:
i do see these in the public API documentation.

http://fullcalendar.io/docs1/utilities/parseISO8601/